### DISCUSSION TITLE
React 18 in faustwp-getting-started example

### DISCUSSION THREAD

**Discussion by Bowriverstudio** [wpengine/faustjs#1090] [Ideas]
It would be nice to see the example using the new features of React

In package.json `https://github.com/wpengine/faustjs/tree/canary/examples/next/faustwp-getting-started`

```json
"react": "^17.0.2",
"react-dom": "^17.0.2",
```

Ideally, I'd prefer to use React 18 - I know the following packages are at v17

```json
"@wordpress/base-styles": "^4.7.0",
 "@wordpress/block-library": "^7.13.0",
```

Thanks

**Comment by Spekachu** [comment]
Yeah I agree. Is it possible to update the dependencies ourselves for a project we create off the example? Or would that not be recommended?
